src/ClientData/CallstackType.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ClientData/CallstackType.h"
+
+#include "OrbitBase/Logging.h"
+
+using orbit_client_protos::CallstackInfo;
+
+namespace orbit_client_data {
+
+std::string CallstackTypeToString(CallstackInfo::CallstackType callstack_type) {
+  switch (callstack_type) {
+    case CallstackInfo::kComplete:
+      return "Complete";
+    case CallstackInfo::kDwarfUnwindingError:
+      return "DWARF unwinding error";
+    case CallstackInfo::kFramePointerUnwindingError:
+      return "Frame pointer unwinding error";
```

Comment:
I'm wondering if we should change "[unwind errors]" to "[Unwind errors]" or "Unwind errors" now that all its subnodes start with a capital letter.